.golangci.yml

```diff
@@ -11,31 +11,30 @@ linters:
     - depguard
     - dogsled
     - errcheck
+    - errorlint
```

Comment:
I alphabetized this list, but also added and removed stuff.
added:
* `errorlint` - makes sure errors are being wrapped and handled correctly.
* `predeclared` - makes sure no predeclared identifiers are being re-declared.
removed:
* `structcheck` - disabled for Go 1.18 due to generics.